src/actions/sso/types.ts

```diff
@@ -13,3 +13,42 @@ export interface IdentitySummary {
   did: string
   publicProfile?: IssuerPublicProfileSummary
 }
+
+export interface AuthenticationRequestSummary {
```

Comment:
The common elements, i.e. `requester`, `callbackURL` could be abstracted into an interface, and then `PaymentRequestSummary`, `AuthenticationRequestSummary`, etc, could extend it.